tiflash/tiflash-supported-pushdown-calculations.md

```diff
@@ -58,3 +58,95 @@ TiFlash supports the following push-down expressions:
+| id                          | estRows | task         | access object | operator info                                                                                    |
++-----------------------------+---------+--------------+---------------+--------------------------------------------------------------------------------------------------+
+| Projection_4                | 4.80    | root         |               | test.t.id                                                                                        |
+| └─Selection_6               | 4.80    | root         |               | lt(cast(time(cast(plus(20230110083056, test.t.a), var_string(20))), var_string(10)), "12:00:00") |
+|   └─TableReader_11          | 6.00    | root         |               | data:ExchangeSender_10                                                                           |
+|     └─ExchangeSender_10     | 6.00    | mpp[tiflash] |               | ExchangeType: PassThrough                                                                        |
+|       └─TableFullScan_9     | 6.00    | mpp[tiflash] | table:t       | keep order:false, stats:pseudo                                                                   |
++-----------------------------+---------+--------------+---------------+--------------------------------------------------------------------------------------------------+
+5 rows in set, 3 warnings (0.20 sec)
+```
+
+The preceding example only performs `TableFullScan` on TiFlash. Other functions are calculated and filtered on `root` and are not pushed down to TiFlash.
+
+You can identify the operators and expressions that cannot be pushed down to TiFlash by running the following command:
+
+```sql
+SHOW WARNINGS;
+
++---------+------+------------------------------------------------------------------------------------------------------------------------------------+
+| Level   | Code | Message                                                                                                                            |
++---------+------+------------------------------------------------------------------------------------------------------------------------------------+
+| Warning | 1105 | Scalar function 'time'(signature: Time, return type: time) is not supported to push down to storage layer now.                     |
+| Warning | 1105 | Scalar function 'cast'(signature: CastDurationAsString, return type: var_string(10)) is not supported to push down to tiflash now. |
+| Warning | 1105 | Scalar function 'cast'(signature: CastDurationAsString, return type: var_string(10)) is not supported to push down to tiflash now. |
++---------+------+------------------------------------------------------------------------------------------------------------------------------------+
+3 rows in set (0.18 sec)
+```
+
+The expressions in the preceding example cannot be completely pushed down to TiFlash, because the functions `Time`and `Cast` cannot be pushed down to TiFlash.
```

Comment:
```suggestion
The expressions in the preceding example cannot be completely pushed down to TiFlash, because the functions `Time` and `Cast` cannot be pushed down to TiFlash.
```